app/upgrades/latest/upgrades.go

```diff
@@ -28,5 +29,9 @@ var Upgrade = upgrades.Upgrade{
 			return migrations, nil
 		}
 	},
-	StoreUpgrades: storetypes.StoreUpgrades{},
+	StoreUpgrades: storetypes.StoreUpgrades{
```

Comment:
do we need to add some form of state to callback?

Reply:
Just the initial values of params. But not sure what values yet.